pkg/operation/botanist/health_check.go

```diff
@@ -213,14 +240,21 @@ func computeRequiredMonitoringStatefulSets(wantsAlertmanager bool) sets.String {
 
 // CheckControlPlane checks whether the control plane components in the given listers are complete and healthy.
 func (b *HealthChecker) CheckControlPlane(
+	gardenerVersion string,
 	shoot *gardencorev1beta1.Shoot,
 	namespace string,
 	condition gardencorev1beta1.Condition,
 	deploymentLister kutil.DeploymentLister,
 	statefulSetLister kutil.StatefulSetLister,
+	etcdLister kutil.EtcdLister,
 	workerLister kutil.WorkerLister,
 ) (*gardencorev1beta1.Condition, error) {
 
+	gardenerVersionLessThan110, err := versionutils.CompareVersions(gardenerVersion, "<", "v1.1.0")
```

Comment:
As just discussed we will merge this PR only after `v1.2.0` release, so this needs to be adapted:

```suggestion
	gardenerVersionLessThan110, err := versionutils.CompareVersions(gardenerVersion, "<", "v1.2.0")
```

Reply:
Also, can you please add a TODO that this code can be removed in a future version again?